Objects/longobject.c

```diff
@@ -3790,6 +3863,42 @@ l_divmod(PyLongObject *v, PyLongObject *w,
     return 0;
 }
 
+/* Compute
+ *     *pmod = mod(v)
+ * NULL can be passed for pmod, in which case the function
+ * never runs code.  The caller owns a reference to
+ * each of these it requests (does not pass NULL for).
+ */
+static int
+l_mod(PyLongObject *v, PyLongObject *w, PyLongObject **pmod)
```

Comment:
It looks like the uses of this function were accidentally removed

Reply:
Oh yeah, I'll have to add them back.